### PR TITLE
Corda flows can do anything blog post

### PR DESCRIPTION
**Pull request by lankydan**

### CODE REVIEW

File: content/blog/2019/corda-flows-can-do-anything/index.md [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+---
+
+In Corda, Flows can do a lot more than proposing new transactions to record between organisations. Although, saying they can do _anything_ is probably a bit far-reaching (it's catchy though). What I really want to say, is that flows are the entry points into a node. Corda provides a series of functions to interact with a node via RPC. These cover the more straightforward use cases, such as querying the vault, but there is a limitation to what is provided. Flows cover any of the _non standard_ logic that needs to be triggered. So, if you want to expose an API from a Corda node that a client can trigger or consume, then this post is for you.
+
+I will be exploring the use of flows as entry points to a node throughout this post. Flows that propose new transactions are shown in many other tutorials and have been excluded from this post. The limelight will be solely shone onto flows that can be leveraged to provide different functionality.
+
+## The theory
+
+As mentioned in the introduction, and I want to reiterate it, the only way to expose an endpoint or API from a Corda node is through a flow. The existing RPC endpoints will handle the _standard_ functionality.
```
Comment (tudor-malene) [inline content/blog/2019/corda-flows-can-do-anything/index.md]: You might want to say that this is the current situation.
It might change in the future.

**Review by tudor-malene — COMMENTED** [review]
Looks good.
Minor comment that this is the current way.
In the future there are plans to create custom cordapp rpc commands.

File: content/blog/2019/corda-flows-can-do-anything/index.md [hunk 2]
```diff
@@ -0,0 +1,150 @@
+---
+title: Flows can do anything
+date: "2019-07-29"
+published: true
+tags: [corda, kotlin, dlt, distributed ledger technology, blockchain]
+cover_image: ./title-card.png
+---
+
+In Corda, Flows can do a lot more than proposing new transactions to record between organisations. Although, saying they can do _anything_ is probably a bit far-reaching (it's catchy though). What I really want to say, is that flows are the entry points into a node. Corda provides a series of functions to interact with a node via RPC. These cover the more straightforward use cases, such as querying the vault, but there is a limitation to what is provided. Flows cover any of the _non standard_ logic that needs to be triggered. So, if you want to expose an API from a Corda node that a client can trigger or consume, then this post is for you.
```
Comment (edward-prosser) [inline content/blog/2019/corda-flows-can-do-anything/index.md]: `non standard` -> `non-standard`

File: content/blog/2019/corda-flows-can-do-anything/index.md [hunk 3]
```diff
@@ -0,0 +1,150 @@
...
+
+I will be exploring the use of flows as entry points to a node throughout this post. Flows that propose new transactions are shown in many other tutorials and have been excluded from this post. The limelight will be solely shone onto flows that can be leveraged to provide different functionality.
```
Comment (edward-prosser) [inline content/blog/2019/corda-flows-can-do-anything/index.md]: `that can be leveraged to provide` -> `that provide`

File: content/blog/2019/corda-flows-can-do-anything/index.md [hunk 4]
```diff
@@ -0,0 +1,150 @@
...
+                externalId = listOf(externalId)
+            )
+        ).states.single().state.data
+    }
+}
+```
+
+If you know your stuff, then you will realise that you can do this via RPC. I chose this example to start you off with something familiar. You have probably done similar queries in your own flows. Although there is a good chance they were part of a more extensive process.
+
+There are a few things to note here that differ compared to most flows you have written:
```
Comment (edward-prosser) [inline content/blog/2019/corda-flows-can-do-anything/index.md]: `to most flows you have written:` -> `to most flows:`

**Pull request merged**